test/framework/kogitodataindex.go

```diff
@@ -15,94 +15,50 @@
 package framework
 
 import (
-	"fmt"
-	"strconv"
-
-	apps "k8s.io/api/apps/v1"
-
 	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
-	"github.com/kiegroup/kogito-cloud-operator/pkg/client/kubernetes"
-	"github.com/kiegroup/kogito-cloud-operator/pkg/framework"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/infrastructure"
 	"github.com/kiegroup/kogito-cloud-operator/test/config"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// InstallKogitoDataIndexService deploy the Kogito Data Index service
-func InstallKogitoDataIndexService(namespace string, installerType InstallerType, replicas int) error {
-	GetLogger(namespace).Infof("%s install Kogito Data Index with %d replicas", installerType, replicas)
-	switch installerType {
-	case CLIInstallerType:
-		return cliInstallKogitoDataIndex(namespace, replicas)
-	case CRInstallerType:
-		return crInstallKogitoDataIndex(namespace, int32(replicas))
-	default:
-		panic(fmt.Errorf("Unknown installer type %s", installerType))
-	}
+// InstallKogitoDataIndex returns the installation service for the Kogito Data Index
+func InstallKogitoDataIndex(namespace string, installerType InstallerType, replicas int) error {
+	serviceName := getDataIndexServiceName()
+	imageTag := getDataIndexImageTag()
+	cliName := "data-index"
+	crBuilder := getDataIndexCrBuilder()
+	persistence := false
```

Comment:
persistence is needed by data-index and will be set automatically
better to put true to avoid confusion

Reply:
If we pass true, then the CLI command will have --enable-persistence which is not valid for Data Index.